### PR TITLE
chore(README): Make required deps a bit more obvious in view-the-docs…

### LINKED ISSUE #2509: Clarify Gulp Dependency in README
#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes)

Requested priority: (Low) 

Products/sites affected: (Fabric UI Repo)

#### Describe the issue:
Even though the "requires gulp" sentence is there, blatantly missed that detail in the install instructions. I think just adding a line item like

1. `npm install -g gulp`
2. `git clone https://github.com/OfficeDev/office-ui-fabric-react.git`
3. `npm install`
4. `npm start`

might make it easier for less observant individuals like myself from ending up with later build errors (that don't mention gulp as a missing dep).

### PR DESCRIPTION
**Pull request by TheLarkInn**
… section

#### Pull request checklist

- [x] Addresses an existing issue: Fixes #2509
- [x] "No change file needed"

#### Description of changes
This updates the README to make installing gulp as a global dependency much more obvious.

#### Focus areas to test
None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,11 +23,14 @@ Fabric React is a responsive, mobile-first collection of robust components desig
...
 
-Before you get started, make sure you have [node.js](https://nodejs.org/), [gulp](http://gulpjs.com/), and [git](https://git-scm.com/) installed. To view the documentation including examples, contracts, component status, and to add functionality or fix issues locally, you can:
+Before you get started, **make sure you have [node.js](https://nodejs.org/) and [git](https://git-scm.com/) installed.**
 
-1. `git clone https://github.com/OfficeDev/office-ui-fabric-react.git`
-2. `npm install`
-3. `npm start`
+To view the documentation including examples, contracts, component status, and to add functionality or fix issues locally, you can:
+
+1. `npm install -g gulp`
```
Comment (micahgodbolt) [inline README.md]: is global gulp actually required? are we not just using a locally installed copy?

Comment (battletoilet) [inline README.md]: if you want to run gulp from the command line then yes, it is required.

Comment (micahgodbolt) [inline README.md]: yeah...guess it doesn't hurt. just saying it's not required to `npm start`

**Comment by TheLarkInn** [comment]
Thanks!!!!

**Pull request merged**